tests/pipeline/test_pipeline.py

```diff
@@ -791,6 +791,56 @@ def test_only_nodes_unknown(self, pipeline_list_with_lists, target_node_names):
         with pytest.raises(ValueError, match=pattern):
             full.only_nodes(*target_node_names)
 
+    @pytest.mark.parametrize(
+        "namespaced_node", ["node1", "node2", "node3", "node4", "node5", "node6"]
```

Comment:
I think this would be better named as ` non_namespaced_node` instead of `namespaced_node`. As the test is trying to pass incorrect node names into a namespaced pipeline and catch the error.

Reply:
Makes sense!